packages/components/MemberList/components/RoleSelector.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TEAM_ROLES } from '@caesar/common/constants';
+import { Select } from '@caesar/components/Select';
+
+const StyledSelect = styled(Select)`
+  width: 170px;
+  height: 32px;
+  border: 1px solid ${({ theme }) => theme.color.gallery};
+  margin-left: auto;
+`;
+
+const OPTIONS = Object.values(TEAM_ROLES).map(role => ({
```

Comment:
this is repeatead code.
lets move it into common/constants like a TEAM_ROLES_OPTIONS.
And import here and in `packages/components/MemberList/components/InviteControl.js`.
What do u think?

Reply:
Done